modules/prompt/functions/async

```diff
@@ -1 +1 @@
-../external/pure/async.zsh
\ No newline at end of file
+../external/async/async.zsh
```

Comment:
Nope, I was still getting errors before I fixed the symlink irrespective of new login shell or not.

After fixing the symlink, I started a new shell to properly load `async` and the `pure` prompt.